### ISSUE TITLE
Interference with custom prompt

### ISSUE THREAD

**Issue by NSxxxxx** [withfig/fig#403]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue

### Issue Details

<img width="644" alt="Снимок экрана 2021-08-29 в 10 13 10" src="https://user-images.githubusercontent.com/80279436/131242108-6c3fa99e-db92-4319-9739-995cc53f41f5.png">
<img width="646" alt="Снимок экрана 2021-08-29 в 10 13 04" src="https://user-images.githubusercontent.com/80279436/131242110-1cc634e1-da24-4d7c-ab18-e043997eb534.png">
<img width="652" alt="Снимок экрана 2021-08-29 в 10 09 22" src="https://user-images.githubusercontent.com/80279436/131242115-268527c1-b12f-4317-9b2b-a9dcc7e85022.png">
<img width="594" alt="Снимок экрана 2021-08-29 в 10 10 04" src="https://user-images.githubusercontent.com/80279436/131242112-0ab66b90-c8d6-4b36-930f-9981d586eb07.png">

Installed oh-my-zsh

### Environment

```bash
Version 1.0.47 (B257) [ABC]
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/ns/.fig/bin/fig
Accessibility: true
Number of specs: 188
SSH Integration: false
Tmux Integration: true
Keybindings path: /Users/ns/.fig/user/keybindings
iTerm Integration: true [Authenticated]
Hyper Integration: false
VSCode Integration: true
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /Users/ns/.nvm/versions/node/v14.17.4/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/zsh (7337) - ttys002
Current working directory: /Users/ns
Current window identifier: 1258/% (com.googlecode.iterm2)
PATH: /Users/ns/.nvm/versions/node/v14.17.4/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/ns/.fig/bin
FIG_INTEGRATION_VERSION: 4
```

**Comment by fwesss** [comment]
@NSxxxxx Is the af-magic prompt for zsh? Looks really familiar and that's something we have an issue with right now.

**Comment by NSxxxxx** [comment]
af-magic is a theme from the oh-my-zsh package

**Comment by fwesss** [comment]
This is fixed in 1.0.49. Run `fig update` to get the fix.
